app/views/online-class/SchedulerView.vue

```diff
@@ -70,11 +71,7 @@ export default {
       classInfo: {},
       time: {},
       studentInfo: {},
-      codeLanguageMap: {
-        python: 'Python',
-        javascript: 'JavaScript',
-        lua: 'Lua'
-      },
+      codeLanguageMap: { ...CODE_LANGUAGE_MAP },
```

Comment:
why copy it? i think use `CODE_LANGUAGE_MAP` directly can also work

Reply:
because I don't want that object to be edited and thus affecting original copy. Since it's in data, in future someone might just set some different value without realizing side-effects